src/public/modules/forms/admin/directives/edit-form.client.directive.js

```diff
@@ -341,9 +341,8 @@ function editFormController(
 
   $scope.isStorageForm = $scope.myform.responseMode === responseModeEnum.ENCRYPT
 
-  // Disable attachment fields when we have webhooks
-  $scope.isDisabledField = function (fieldType) {
-    return fieldType.name === 'attachment' && $scope.myform.webhook.url !== ''
+  $scope.isDisabledField = function () {
+    return false
```

Comment:
can we simply remove this function entirely?

Reply:
Sure, but I'll do it in a separate PR?